benches/hash.rs

```diff
@@ -1,9 +1,13 @@
 use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
 use miden_crypto::{
-    hash::rpo::{Rpo256, RpoDigest},
+    hash::{
+        blake::Blake3_256,
+        rpo::{Rpo256, RpoDigest},
+    },
     Felt,
 };
 use rand_utils::rand_value;
+use winter_crypto::Hasher;
```

Comment:
@bobbinth should we also get this here from `miden-core` as your feedback was on the delendum benchmarking RP?

Reply:
Not here as `miden-core` will likely depend on this module in the future (rather than the other way around).